tests/cypress/integration/IKUpload.cy.js

```diff
@@ -0,0 +1,39 @@
+describe('IKUpload Element', () => {
```

Comment:
This PR fixes this bug - https://github.com/imagekit-developer/imagekit-react/issues/71
Should we not add one more test case in `IKImage.cy.js` to assert the case which was causing this issue in the first place.
In test case we need to use `<IKImage lqip={{ active: false }} {...other} />` and ensure that image loads.